test/es-module/test-esm-experimental-warnings.mjs

```diff
@@ -10,7 +10,7 @@ describe('ESM: warn for obsolete hooks provided', { concurrency: true }, () => {
     const { code, signal, stderr } = await spawnPromisified(execPath, [
       '--input-type=module',
       '--eval',
-      `import ${JSON.stringify(fileURL('es-module-loaders', 'module-named-exports.mjs'))}`,
+      `import '${fileHref('es-module-loaders', 'module-named-exports.mjs')}'`,
```

Comment:
ditto

Reply:
Why?

Reply:
> Why?

The full path could contain apostrophes, which wouldn’t be escaped in this new version (but were in the previous with `JSON.stringify`). Imagine something like `/Volumes/C/Program Files/Antoine's Projects/node/test/fixtures/es-module-loaders/module-named-exports.mjs`.